actor/src/main/scala/org/apache/pekko/serialization/Serialization.scala

```diff
@@ -466,8 +462,12 @@ class Serialization(val system: ExtendedActorSystem) extends Extension {
     }
   }
 
+  @nowarn("msg=deprecated")
   private def warnUnexpectedNonPekkoSerializer(clazz: Class[_], ser: Serializer): Boolean = {
-    if (clazz.getName.startsWith("org.apache.pekko.") && !ser.getClass.getName.startsWith("org.apache.pekko.")) {
```

Comment:
I don't think it is good idea to turn off these warnning, if have to, better using same way like `shouldWarnAboutJavaSerializer` rather than using a list struct.